### DISCUSSION TITLE
Allow JMX Beans management actions

### DISCUSSION THREAD

**Discussion by Kusig** [javamelody/javamelody#1186] [Ideas]
Currently only JMX Beans values are shown but management actions could not be executed.
It would be very useful if the execution of JMX Beans management actions are possible as well from JavaMelody JMX Beans viewer.

Additionally, having the opportunity to as well execute the management actions as uri operation would be appreciated (similar as the current GET functionality but with a POST invoked for example).

**Comment by evernat** [comment]
It is not planned at the moment. From a security point of view, JMX MBeans write is quite different than JMX MBeans read or than metrics read. Still, there is an issue for that if you want to vote : #297

**Comment by Kusig** [comment]
Thanks, voted there and added comment about.

However, one must protect Javamelody anyhow and there is already sensible information available on JMXBeans anyhow. So I don't see an additional security risk.

Just make it configurable to be activated/deactivated and problem solved :-)
